jobs/webcompat-kb/webcompat_kb/main.py

```diff
@@ -39,6 +65,26 @@
         "o3": "isempty",
         "f4": "CP",
     },
+    "site_reports_other": {
+        "f1": "product",
+        "o1": "notequals",
+        "v1": "Web Compatibility",
+        "f2": "keywords",
+        "o2": "substring",
+        "v2": "webcompat%3Asite-report",
+    },
+    "knowledge_base_wc": {
+        "product": "Web Compatibility",
+        "component": "Knowledge Base",
+    },
+    "knowledge_base_other": {
+        "f1": "product",
+        "o1": "notequals",
+        "v1": "Web Compatibility",
+        "f2": "keywords",
+        "o2": "substring",
+        "v2": "webcompat%3Aplatform-bug",
```

Comment:
```suggestion
        "v2": "webcompat:platform-bug",
```
It doesn't seem to return anything when using URL-encoded colon, looks like requests library encodes it again, so it ends up double encoded